dxr/utils.py

```diff
@@ -295,7 +295,16 @@ def split_content_lines(unicode):
     resulting lines.
 
     """
-    return unicode.splitlines(True)
+    lines = unicode.splitlines(True)
+    # Vertical Tabs and Form Feeds are treated as end-of-lines by splitlines
```

Comment:
I ran the contents of Debugger.cpp (from the below linked bug) through the original version of my function from this PR as well as a version that does `[line.decode('utf-8') for line in unicode.encode('utf-8').splitlines(True)]`.  The original version takes ~5ms, the encode-and-decode version takes ~8ms.

I can also modify the original version to use `... in (u"\v", u"\f", u"\x1c", ...)` to cover all the oddball whitespace characters.  This actually runs just a bit faster than my original version at ~4.6ms.

Tell me what you'd prefer.